web/modules/custom/collection/src/EventSubscriber/CollectionEventSubscriber.php

```diff
@@ -0,0 +1,56 @@
+<?php
+
+namespace Drupal\collection\EventSubscriber;
+
+use Drupal\og\Event\PermissionEventInterface;
+use Symfony\Component\EventDispatcher\EventSubscriberInterface;
+
+/**
+ * Event subscribers for the Collection module.
+ */
+class CollectionEventSubscriber implements EventSubscriberInterface {
+
+  /**
+   * {@inheritdoc}
+   */
+  public static function getSubscribedEvents() {
+    return [
+      PermissionEventInterface::EVENT_NAME => [['provideDefaultOgPermissions']],
+    ];
+  }
+
+  /**
+   * Provides default OG permissions.
```

Comment:
I'm a bit sleepy today (meeting overflow), so this might have something to do with it.
I didn't get it straight away that permissions are declared here.
Maybe something like: "Declare the permissions defined for collections." could help.
(Although that sentence doesn't seem grammatically correct. I can't come up with anything else at this point though. Maybe after a proper night of sleep.)